server/app/services/program/BlockDefinition.java

```diff
@@ -225,6 +225,18 @@ public int getQuestionCount() {
     return programQuestionDefinitions().size();
   }
 
+  /**
+   * Returns true if any of the question definitions in this block are QuestionType.NULL_QUESTION
+   */
+  @JsonIgnore
+  @Memoized
+  public boolean hasNullQuestion() {
```

Comment:
**nitpick:** add a unit test case in BlockDefinitionTest

Reply:
Already had them for this